Makefile

```diff
@@ -1,20 +1,26 @@
-.PHONY: build test run clean stop check-style run-unit emojis help
+.PHONY: build test run clean stop check-style run-unit emojis help test-ci
 
 BUILD_SERVER_DIR = ../mattermost-server
 BUILD_WEBAPP_DIR = ../mattermost-webapp
 MM_UTILITIES_DIR = ../mattermost-utilities
 EMOJI_TOOLS_DIR = ./build/emoji
 
-check-style: node_modules ## Checks JS file for ESLint confirmity
```

Comment:
I think this changes can affect to the developers workflow. The node_modules dependency is important there. Maybe we can use in the circusci configuration the `npm run XXX` commands instead of the `make XXX` commands.

Reply:
sounds good will update

Reply:
done